R/bm-dataset-taxi-parquet.R

```diff
@@ -87,6 +87,14 @@ dataset_taxi_parquet <- Benchmark("dataset_taxi_parquet",
           identical(sum(result$n), 3069271L)
         )
       }
+    ),
+    count_rows = list(
+      query = function(ds) {
+        dim(ds)
+      },
+      assert = function(result) {
+        stopifnot(identical(result, c(1547741381L, 20L)))
```

Comment:
Minor, and you don't have to take it if you don't want, but this will provide a slightly more informative message if the assertion fails. (Also, yes this syntax is strange, don't ask me how it got to be this way)

```suggestion
        stopifnot("dims does not match" = identical(result, c(1547741381L, 20L)))
```